include/z64animation.h

```diff
@@ -37,7 +37,7 @@ typedef enum AnimationMode {
     /* 5 */ ANIMMODE_LOOP_PARTIAL_INTERP
 } AnimationMode;
 
-typedef enum {
+typedef enum AnimationTapers {
```

Comment:
```suggestion
typedef enum AnimationTaper {
```
Could you also update the name at the bottom of the enum to be singular as well?